kubeai/README.md

```diff
@@ -190,19 +190,19 @@ Enjoy the answer!
 ## Overview
 
 [NRI plugins][nri-plugins] provide a way to
-optimize the resource placement of applications in a Kubernetes cluster. They
-connect to the container runtime and are able, for example, to adjust the CPU
-and memory pinning of containers.
+optimize the node-level resource assignment of applications in a Kubernetes
+cluster. They connect to the container runtime and are able, for example, to
+adjust the CPU and memory pinning of containers.
 
 This section provides a guide on how to use the
 [Balloons Policy][balloons-policy] plugin from the [NRI Plugins][nri-plugins]
 project to optimize the performance of CPU-backed KubeAI profiles.
 
 ## Installation of Balloons Policy Plugin
 
-> **NOTE:** To avoid disturbing already running workloads it is recommended to
-> install the NRI plugin to an empty node (do it right after node bootstrap, or
-> drain the node before installation).
+> **NOTE:** To avoid disturbing already running workloads, it is recommended to
+> install the NRI plugin on an empty node, that is, before deploying workloads
```

Comment:
Admin does not need to install the plugin on the node itself, but can do it using Kubernetes facilities from outside the node, to install it **to** the node, right?

Reply:
Yeah, maybe that's better, "onto"?

Reply:
@marquiz Are you going to update the PR?  (I can merge it after that as Sakari has already approved this.)